### DISCUSSION TITLE
Get Recommendations with Target/Min/Max

### DISCUSSION THREAD

**Discussion by CraftCodeYS** [JohnnyCrazy/SpotifyAPI-NET#799] [Q&A] [answered]
Hello everyone,
I would like to get a list of recommendations with specific target values for audio features. In the metadata you can see the dictionary in the Request. But when you make an instance of this object, you can´t set the keys and values. Do you have any ideas how to wirte the right syntax for an dicitionary in this situation? 

![C#-SpotifyApi-GetRecommendations-Metadata](https://user-images.githubusercontent.com/22844052/189630808-4fc81a03-45ab-4db9-98d5-8d02c4fb5eaa.PNG)
![C#-SpotifyApi-GetRecommendations](https://user-images.githubusercontent.com/22844052/189630275-a6c39645-fdf9-45e4-97c7-32757b0fce50.PNG)

**Comment by CraftCodeYS** [comment]
this is the documentation of spotify api: 
https://developer.spotify.com/documentation/web-api/reference/#/operations/get-recommendations

In this picture you can see an example for a https request (this is working):
![Https_Anfrage](https://user-images.githubusercontent.com/22844052/189632452-4416ebcf-4dc8-4889-99d8-a810d767ed22.PNG)

**Comment by JohnnyCrazy** [comment, marked as answer]
You can use the `Add` methods on the dictionary, e.g:

```c#
var request = new RecommendationsRequest();
request.Target.Add("energy", "0.9");
request.Target.Add("instrumentalness", "0.75");
```

**Comment by CraftCodeYS** [comment]
Awesome! This works, thank you so much!
